assets/js/img_clicker.js

```diff
@@ -0,0 +1,23 @@
+$(function () {
+    "use strict";
+
+    $(".containerTeam .imgMembers").click(function () {
+        var $src = $(this).attr("src"); 
+        var $description = $(this).siblings("#desc").text();
+        var $name= $(this).siblings("h4").text();
```

Comment:
Missing space before equals sign. This file has a few style issues like this one.